README.md

```diff
@@ -1,122 +1,120 @@
-# Andrew-Chen-Wang/github-wiki-action
-Updates your GitHub wiki by using rsync.
-
-This action updates your repository's wiki
-based on a single directory that matches with
-your Wiki's git. You can use a Wiki directory
-from any repository you wish.
-
-_**It is recommended that you still have a Home.md
-or whatever extension you want instead of MD.**_ This
-is so that GitHub doesn't automatically make a Home.md
-for you again.
-
-Table of Contents:
-- [Features](#features)
-- [Usage](#usage)
-- [Inputs](#inputs)
-- [Inspiration](#inspiration)
-- [License](#license)
-- [Non-Affiliation with Github Inc.](#non-affiliation-with-github-inc)
-
----
-### Features
-
-- rsync all your files from one directory (either from the current or other repository) to your GitHub's repo's wiki.
-    - rsyncing from a different repository requires a [GitHub PAT](https://github.com/settings/tokens/new?scopes=repo&description=wiki%20page%20creator%20token)
-    - If you use a private repository, you may have to use a GitHub PAT.
-- Use the commit message from your repository's git's commit. You can specify a custom one if you want.
-- Be able to exclude files and directories based on an input of a list.
-
----
-### Usage
-
-You must have a single wiki page available from the beginning.
-It can be blank, but there must be at least one page that exists.
-You must also have a directory where all your wiki files will
-be located (the default directory is "wiki/"). To include the
-mandatory homepage, have a file in your wiki/ directory
-called Home.md or with any other extension (e.g. rst).
-
-```yaml
-name: Deploy Wiki
+# Publish to GitHub wiki
 
+📖 GitHub Action to sync a folder to the GitHub wiki
+
+📂 Keep your dev docs in sync with your code \
+🔁 Able to open PRs with docs updates \
+🗂️ Use the fancy GitHub wiki reader view \
+💡 Inspired by [Decathlon/wiki-page-creator-action#11]
+
+**GitHub Wiki Action** is not certified by GitHub. It is provided by a
+third-party and is governed by separate terms of service, privacy policy, and
+support documentation.
```

Comment:
really i don't even know why its needed since it appears on the page sidebar as a disclaimer added by github. i just copied that into this repo since I figured you wanted it 🤷

![image](https://user-images.githubusercontent.com/61068799/227314350-7e3ec168-2928-4d8d-89e4-6989d92db190.png)

https://github.com/marketplace/actions/github-wiki-action

Reply:
i don't think this is required because literally no other action does it. it's only there on github's marketplace to indicate that the github.com skin thing with github branding and stuff doesn't mean they endorse _your product_.

you don't need that notice in your readme though since that's already covered under other terms/conds as being _yours_ not _githubs_ (i think?)